managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedCursorContainerTest.java

```diff
@@ -386,6 +389,31 @@ public boolean checkAndUpdateReadPositionChanged() {
         }
     }
 
+    @Test
+    public void testSlowestReadPositionForActiveCursors() throws Exception {
```

Comment:
Better to test the cursor's read position move forward case?

Reply:
Good catch. I've fixed it